pkg/hubble/exporter/dynamic_exporter_test.go

```diff
@@ -31,6 +31,18 @@ func TestDynamicExporterLifecycle(t *testing.T) {
 	exporterConfigParser := &exporterConfigParser{logger}
 	dynamicExporter := NewDynamicExporter(logger, fileName, exporterFactory, exporterConfigParser)
 
+	// Start watching to trigger initial config load
+	ctx, cancel := context.WithCancel(context.Background())
```

Comment:
```suggestion
	ctx, cancel := context.WithCancel(t.Context())
```
todo: Pass the testing context to propagate test cancellation.

Reply:
Redundant now.